dvc/remote/local.py

```diff
@@ -591,12 +597,16 @@ def status(self, checksum_infos, remote, jobs=None, show_checksums=False):
         progress.finish_target(title)
 
         for md5, info in ret.items():
-            info["status"] = STATUS_MAP[
-                (md5 in local_exists, md5 in remote_exists)
-            ]
+            self._fill_status(info, md5, local_exists, remote_exists)
 
         return ret
 
+    def _fill_status(self, info, md5, local_exists, remote_exists):
+        status = STATUS_MAP[(md5 in local_exists, md5 in remote_exists)]
+        if status == STATUS_BROKEN:
+            raise NoChecksumInLocalAndRemoteCache(info["name"], md5)
```

Comment:
Seems like raising an exception on the first missing cache file is a little too intrusive. Maybe we could use a warning instead? After all, even if some cache is missing(e.g. someone forgot to push one of models), it is not the end of the world and we still should pull all we can. What do you think?

Reply:
Sure, that might be exaggeration.

Reply:
@pared Btw, maybe it is also worth collecting all broken cache first and then priting it in one warning, so that we don't flood the terminal with the same message :)

Reply:
Sure!